documentation/architecture/desktop_application/data-model.md

```diff
@@ -0,0 +1,80 @@
+# Desktop application
+
+This document presents the data model used in the desktop application.
+
+There are 3 main activities available to a user when using the application:
+
+1. Enroll / Reset / Quit
+2. Share a local service / Invite a user to a shared service
+3. Accept an invitation / Connect to a remote service
+
+The following sections show the main data types supporting the scenarios above
+
+# Enrollment
```

Comment:
Rather than earmark a bunch of spots, I would probably capitalize `Space`, `Project`, `Project Node` and perhaps "invitation" - let me know if you think the slight informality is preferable in this context.

Reply:
I tend to use the informal capitalization unless I refer to something more formal, in a diagram, or in some code.